Source/WTF/wtf/Compiler.h

```diff
@@ -270,7 +270,7 @@
 
 /* LIKELY */
 
-#if !defined(LIKELY) && COMPILER(GCC_COMPATIBLE)
+#if !defined(LIKELY)
 #define LIKELY(x) __builtin_expect(!!(x), 1)
```

Comment:
Don't we need to remove the `#if !defined(LIKELY)` block below (and likewise for the various defines that follow)?

Reply:
We are keeping it since we may want to inject `LIKELY` definition outside of this file.

Reply:
I think Ross is talking about the definition right below here:

```#if !defined(LIKELY)
#define LIKELY(x) (x)
#endif```

Reply:
That is indeed what I meant; there appears to be lots of unreachable code created by this patch.